mesonbuild/dependencies/misc.py

```diff
@@ -39,6 +39,9 @@ class BoostDependency(ExternalDependency):
 
     def __init__(self, environment, kwargs):
         super().__init__('boost', environment, 'cpp', kwargs)
+        # Building boost for static linking is default in windows. Is this even possible?
+        #if mesonlib.is_windows() and 'static' not in kwargs.keys():
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->